darwin/future/meta/queries/item_id.py

```diff
@@ -0,0 +1,41 @@
+from functools import reduce
+from typing import Dict
+
+from darwin.future.core.items.get import get_item_ids
+from darwin.future.core.types.common import QueryString
+from darwin.future.core.types.query import PaginatedQuery
+from darwin.future.meta.objects.v7_id import V7ID
+
+
+class ItemIDQuery(PaginatedQuery[V7ID]):
+    def _collect(self) -> Dict[int, V7ID]:
+        if "team_slug" not in self.meta_params:
+            raise ValueError("Must specify team_slug to query item ids")
+        if (
+            "dataset_ids" not in self.meta_params
+            and "dataset_id" not in self.meta_params
+        ):
+            raise ValueError("Must specify dataset_id to query item ids")
+        team_slug: str = self.meta_params["team_slug"]
+        dataset_ids: int = (
+            self.meta_params["dataset_ids"]
+            if "dataset_ids" in self.meta_params
+            else self.meta_params["dataset_id"]
+        )
+        params: QueryString = reduce(
```

Comment:
How I use the aforementioned QueryString + QueryString syntax, map reduce them with the page object and all the filters

Reply:
I like that python has `itertools`, but I'd prefer a fluent interface like any sensible language ~like Rust~

Anyway, good stuff!